src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -100,6 +113,28 @@ public boolean Disconnect(String peerId){
 
         return ret;
     }
+
+    //test time implementation to simulate 'peer disappearing'
+    public boolean  DisconnectIncomingConnections(){
```

Comment:
A big part of this function's purpose was testing how the native layer handles connections just dying. So I was expecting us to do something really nasty like jump down to the bluetooth server socket object and call close on it and see how the rest of the native code handles that. The kill below is way too clean to really test anything at the native layer.